src/citrine/informatics/executions/sample_design_space_execution.py

```diff
@@ -28,9 +28,20 @@ def _path(self):
     @classmethod
     def _pre_build(cls, data: dict) -> dict:
         """Run data modification before building."""
-        status = data.get('status')
-        update_dict = {"status": status["minor"], "status_detail": status["detail"]}
-        data.update(update_dict)
+        status_dict = data['status']
+
+        # Set major status
+        if "major" not in status_dict and status_dict["minor"] in ["EXECUTING", "VALIDATING"]:
```

Comment:
I think we always want to set `status_dict['major']` based on the value of `status_dict['minor']` if the server doesn't. That way, no matter the status, the interface matches the other executions.